source/documentation/index.md

```diff
@@ -27,15 +27,13 @@ When you are ready to test your software, first read and understand the [CTC Tra
 
 We strongly advise you to start testing your software for compatibility purposes as soon as possible.
 
-## What is Trader Test?
+## What is the CTC Traders Test Support API?
 
-**Before you can get access to the live production environment, you must check that your software is fully compatible with our CTC Traders API.**
+The CTC Traders Test Support API enables testers to inject transit movement notifications as if they have been sent by the New Computerised Transit System (NCTS) from an office of departure or an office of destination. Until automated responses to trader messages are supported in a trader test environment, the test support API allows for responses to be triggered manually.
```

Comment:
Is the description supposed to explain what the Trader Support API isn't? Is there agreement on that? I've asked Tim Squires about the issue (awaiting reply).

Reply:
I've added a note to say it's not a substitute for Trader Test and is due to be switched off when Trader Test becomes available.